unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldBridge/BuilderInWorldBridge.cs

```diff
@@ -104,16 +112,19 @@ public void EntityTransformReport(DecentralandEntity entity, ParcelScene scene)
 
         WebInterface.SceneEvent<ModifyEntityComponentEvent> sceneEvent = new WebInterface.SceneEvent<ModifyEntityComponentEvent>();
         sceneEvent.sceneId = scene.sceneData.id;
-        sceneEvent.eventType = "stateEvent";
+        sceneEvent.eventType = BuilderInWorldSettings.STATE_EVENT_NAME;
         sceneEvent.payload = modifyEntityComponentEvent;
 
-        string messasage = JsonConvert.SerializeObject(sceneEvent, Formatting.None, new JsonSerializerSettings
+
+        //Note (Adrian): We use Newtonsoft instead of JsonUtility because we need to deal with super classes, JsonUtility doesn't enconde them
```

Comment:
```suggestion
        //Note (Adrian): We use Newtonsoft instead of JsonUtility because we need to deal with super classes, JsonUtility doesn't encode them
```